File: p4/p4.py
def ans():
    maxpal = 0
    for i in range(999, 100, -1):
        for j in range(999, 100, -1):
            if str(i*j) == str(i*j)[::-1] and i*j > maxpal:
                maxpal = i*j
    return maxpal

File: p4/p4.jl
function answer()
	maxpal = 0
	for i = 999:-1:100
		for j = 999:-1:100
			s = string(i*j)
			if s == s[end:-1:1]  && i*j > maxpal
				maxpal = i*j
			end	
		end	
	end	
	return maxpal
end	
